### DISCUSSION TITLE
Security

### DISCUSSION THREAD

**Discussion by arcticfly** [TaxyAI/chrome-extension#26] [General]
We know that security is a big deal in the AI space. Right now, Taxy shouldn't be left unsupervised. Not only can it run up your OpenAI bill, but it can also make changes you didn't expect. This is a place for the community to discuss what additions they think would be helpful to ensure that Taxy is safe to use for everyone.

**Comment by Youssef535** [comment]
However, I can tell you that any application that uses AI technology or processes user data must follow strict data protection rules and regulations. Additionally, developers must ensure the security of the data they gather and process, and users must be aware of what data is being collected and how it is being used. It is essential to use caution while sharing personal information online and to trust only reliable and reputable applications with data privacy policies that are transparent and comprehensive.
